addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -335,50 +323,40 @@ func _submit_full_state(received_state: Dictionary, received_tick: int):
 		
 		sanitized[property_name] = value
 	
-	# Duplicates the previous state(s), including the current one
-	# Also fixes packet loss (e.g. a state missing)
-	if (NetworkRollback.enable_state_diffs && _latest_state_tick != -1):
-		for picked_tick in range(_latest_state_tick, received_tick - 1):
-			_states[picked_tick + 1] = _states[picked_tick].duplicate(true)
```

Comment:
Enabling diff states, it often gets inside this condition with any packet loss (kinda often), and with this, you can gap missing states of the past. There is no reason to not have this, as it ensures smoothness imo.